src/main/scala/is/hail/utils/LocalVariantSortIterator.scala

```diff
@@ -1,72 +0,0 @@
-package is.hail.utils
```

Comment:
I guess this isn't used any more? I'm not clear on how it relates to implicit orderings.

Reply:
It wasn't used anymore so Cotton said to delete it.